A3-Antistasi/fastTravelRadio.sqf

```diff
@@ -31,7 +31,7 @@ if (_chequeo) exitWith {Hint "You cannot Fast Travel with enemies near the group
 	}
 } forEach units _grupo;
 
-if (_chequeo) exitWith {Hint "You cannot Fast Travel if you don't have a driver in all your vehicles or your vehicles are damaged and cannot move or your group is in a boat"};
+if (_chequeo) exitWith {Hint "You cannot Fast Travel if your vehicles don't have a driver, your vehicles immobile or your group is in a boat"};
```

Comment:
what do you think of `your vehicles is immoblized ...` ?

Reply:
yeah, that sounds better. Though replace "is" with "are" :)

Reply:
Should it be plural? I thought fast travel only affected on vehicle at a time